### ISSUE TITLE
bug: The base os image version is not correct

### ISSUE THREAD

**Issue by bonito-skipjack** [tensorchord/envd#1240]
## Description
In build.envd states ubuntu 22.04 as base os, but after start up the container, the os is ubuntu 20.04
Inside container execute cat /etc/issue  got beneath output
Ubuntu 20.04.5 LTS \n \l
## Reproduction
envd up
## Additional Info
envd version
envd: v0.2.5-rc.1
  BuildDate: 2022-11-28T09:07:39Z
  GitCommit: 69fe6ce6ffa15deba958fd3252137dab2a62951e
  GitTreeState: clean
  GitTag: v0.2.5-rc.1
  GoVersion: go1.18.8
  Compiler: gc
  Platform: linux/amd64

cat build.envd
def build():
    base(os="ubuntu22.04", language="python3.8")
    # Configure pip index if needed.
    # config.pip_index(url = "https://pypi.tuna.tsinghua.edu.cn/simple")
    install.python_packages(name = [
        "numpy",
    ])
    shell("bash")

**Comment by xieydd** [comment]
```
root in aiops_algorithm_service on  hidveghuang_dev via Py v3.9.15 as sudo
⬢ [envd]❯ cat /etc/issue
Ubuntu 20.04.5 LTS \n \l
```

Do have this problem.

**Comment by kemingy** [comment]
Yep. It's hard-coded. Will be fixed in #1228.

**Comment by kemingy** [comment]
We are able to support it now. But the new syntax lack docs. You may refer to the test [examples](https://github.com/tensorchord/envd/blob/main/e2e/v1/docs/testdata/complex/build.envd).

We'll add the docs later.
